src/Workspaces/Core/Portable/Classification/ClassificationTypeNames.cs

```diff
@@ -11,11 +11,24 @@ public static class ClassificationTypeNames
         /// <summary>
         /// Additive classifications types supply additional context to other classifications.
         /// </summary>
-        /// <remarks>
-        /// NOTE: Any updates to this class should also be reflected in LSP - see
-        /// Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensHelpers.cs
-        /// </remarks>
-        public static ImmutableArray<string> AdditiveTypeNames { get; } = ImmutableArray.Create(StaticSymbol);
+        public static ImmutableArray<string> AdditiveTypeNames { get; } = ImmutableArray.Create(StaticSymbol, ReassignedVariable);
```

Comment:
Curious: was `ReassignedVariable` a piece we had previously missed on the LSP side that we noticed / added here after the test caught the mismatch?